### DISCUSSION TITLE
Tailwind Child Theme or not?

### DISCUSSION THREAD

**Discussion by sperlari** [tailpress/tailpress#199] [Q&A] [answered]
Hi everyone,
Is it recommended or not to create a child theme with this theme? And why?
I read some previous answers but I noticed that a child theme was created but it doesn't work for me...

Thanks for the clarifications

**Comment by jeffreyvr** [comment]
Hi,

I personally use the theme as a starting point and haven't yet made a child theme with it.

However, if you'd like to make a child theme, you might get inspiration from this setup.

https://github.com/jeffreyvr/tailpress-child-theme-demo

If it doesn't work for you, maybe you can provide more info an what doesn't work as we can have a look.

**Reply by sperlari** [reply]
Thanks for the reply. It gives me these errors:

2 ERRORS in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
webpack compiled with 4 errors

ERROR in ./resources/css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
Error: Failed to find 'tailwindcss/base

ERROR in ./resources/css/editor-style.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
Error: Failed to find 'tailwindcss/components'

Question that may be stupid and I apologize for that:

I have to proceed with npm install and npm run watch also in the parent theme..is this the case?
Thanks

**Comment by jeffreyvr** [comment, marked as answer]
> I have to proceed with npm install and npm run watch also in the parent theme..is this the case?

This appears to be the case. Not ideal, but at the moment, I don't know how to get it working without it.

**Reply by sperlari** [reply]
Yes, it's not ideal but now it works...actually as you said, it's also ok to keep the parent theme and that's it. Anyway, it works and so everything is ok!

Thank you!
